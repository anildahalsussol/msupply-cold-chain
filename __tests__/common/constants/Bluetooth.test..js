import { BLUETOOTH } from '~constants';

describe('Constants: BLUETOOTH', () => {
  it('Matches snapshot', () => {
    expect(BLUETOOTH).toMatchSnapshot();
  });
});
