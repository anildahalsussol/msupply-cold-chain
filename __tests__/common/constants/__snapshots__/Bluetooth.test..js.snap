// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: BLUETOOTH Matches snapshot 1`] = `
Object {
  "READ_CHARACTERISTIC_UUID": "6e400002-b5a3-f393-e0a9-e50e24dcca9e",
  "UART_SERVICE_UUID": "6e400001-b5a3-f393-e0a9-e50e24dcca9e",
  "WRITE_CHARACTERISTIC_UUID": "6e400003-b5a3-f393-e0a9-e50e24dcca9e",
}
`;
